src/moin/translations/ru/LC_MESSAGES/messages.po

```diff
@@ -181,7 +181,7 @@ msgstr ""
 #: src/moin/apps/admin/views.py:83
 #, fuzzy, python-format
 msgid "User profile of %(username)s: %(email)s %(disabled)s"
-msgstr "Профиль пользователя %(username)s: %(email)r"
+msgstr "Профиль пользователя %(username)s: %(email)s"
```

Comment:
disabled placeholder is still missing.

Reply:
good that you let that "fuzzy" there. likely some strings are still wrong, it just makes the warnings at install time go away.